crates/ruff_formatter/src/token/number.rs

```diff
@@ -1,296 +0,0 @@
-use crate::token::string::ToAsciiLowercaseCow;
-use ruff_rowan::{Language, SyntaxToken};
-use std::borrow::Cow;
-use std::num::NonZeroUsize;
-
-use crate::prelude::*;
-use crate::{CstFormatContext, Format};
-
-pub fn format_number_token<L>(token: &SyntaxToken<L>) -> CleanedNumberLiteralText<L>
```

Comment:
We may use some of this (together with `string`) to normalize the string/number content. But we can pull this in when needed.

Reply:
Yeah we'll absolutely need this kind of logic. But does it belong here, or in `ruff_python_formatter`? Black has rules around this stuff, but some of it is Python-specific (e.g., for complex numbers).